app/javascript/controllers/todos_controller.js

```diff
@@ -0,0 +1,11 @@
+import { Controller } from 'stimulus';
+
+export default class extends Controller {
+  static targets = ['toggleAll', 'checkbox'];
+
+  toggleAll() {
+    this.checkboxTargets.forEach(function(element) {
+      element.click();
```

Comment:
If this is for the functionality I'm thinking, we actually need to detect whether or not any are unchecked. If so then we check all; otherwise, we uncheck all.